content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,146 @@
+title: Empower your documentation with addons
+date: 2024-03-27
+category: Changelog
+tags: addons, hosting
+authors: Manuel Kaufmann
+status: published
+image: /images/enable-beta-addons.jpg
+image_credit: Photo by <a href="https://unsplash.com/@impatrickt?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Patrick Tomasso</a> on <a href="https://unsplash.com/photos/open-book-lot-Oaqk7qqNh_c?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+In the last year, we have been focusing ourselves on providing a better experience for authors and readers of the documentation.
+Today, we are thrilled to introduce our latest feature,
+designed on top of users' feedback to build a better experience for them in our platform: **Read the Docs Addons**.
+
+Read the Docs Addons helps authors to keep their documentation updated by focusing on the most relevant pages,
+while simplifying the review process --and also helps readers to navigate the documentation in a simple way,
+always knowing the versions they are reading avoiding confusions.
+
+Continue reading to explore the full list of addons and how to enable them in your project.
+
+## Full list of addons
+
+<!--
+TODO describe each of the addons into its own section,
+similar to what we have in https://github.com/readthedocs/addons/blob/main/README.md
+
+Consider also https://about.readthedocs.com/docs-as-code/
+-->
+
+### Analytics
+
+![Analytics Addons](/images/addons-analytics.png)
+
+Analytics allows you to explore your documentation traffic to understand what are the pages your readers read the most.
```

Comment:
```suggestion
Analytics allows authors to explore your documentation traffic to understand what are the pages readers read the most.
```

I generally like referring to "you" in blogs but it's a little unclear here and at some other points in the blog. We are referring to you as a documentation author/manager here while above we refer to "authors". We should be consistent. If we think non-authors will read the blog, maybe just use "authors" and "readers".